nsxt/resource_nsxt_policy_ldap_identity_source.go

```diff
@@ -219,6 +220,9 @@ func resourceNsxtPolicyLdapIdentitySourceProbeAndUpdate(d *schema.ResourceData,
 			LdapServers:            ldapServers,
 			ResourceType:           nsxModel.LdapIdentitySource_RESOURCE_TYPE_ACTIVEDIRECTORYIDENTITYSOURCE,
 		}
+		if util.NsxVersionLower("4.2.1") {
```

Comment:
This does not seem to change behavior because DisplayName is still assigned in line 212

Reply:
Thanks, it's fixed